nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,18 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 1.0.2
+***************
+
+* Implement RAI (Release Assistance Indication) support in modem lib.
+* Fixed issue with socket state machine so that a RX event is not triggered on a socket closed by peer. Which would trigger both a POLLIN and a POLLHUP on socket closed by peer.
```

Comment:
`Fixed an issue that would lead to both POLLIN and POLLHUP being reported by poll() when a connection is closed by the peer`

Reply:
ok